<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1337</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>目标跟踪方法调研</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/logo/huge/logo-huge.jpg</normaloff>:/logo/huge/logo-huge.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="widget_4" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QSplitter" name="splitter_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QWidget" name="widget_10" native="true">
                  <layout class="QGridLayout" name="gridLayout_11">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label">
                     <property name="maximumSize">
                      <size>
                       <width>54</width>
                       <height>12</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>图源获取</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_11" native="true">
                  <layout class="QGridLayout" name="gridLayout_12">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton">
                     <property name="maximumSize">
                      <size>
                       <width>75</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>打开序列</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_12" native="true">
                  <layout class="QGridLayout" name="gridLayout_13">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton_2">
                     <property name="maximumSize">
                      <size>
                       <width>75</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>打开视频</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_13" native="true">
                  <layout class="QGridLayout" name="gridLayout_14">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton_3">
                     <property name="maximumSize">
                      <size>
                       <width>75</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>打开摄像头</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="widget_5" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QSplitter" name="splitter_5">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QWidget" name="widget_14" native="true">
                  <layout class="QGridLayout" name="gridLayout_15">
                   <item row="0" column="0">
                    <widget class="QLabel" name="label_2">
                     <property name="maximumSize">
                      <size>
                       <width>54</width>
                       <height>12</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>初始化</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_15" native="true">
                  <layout class="QGridLayout" name="gridLayout_16">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton_4">
                     <property name="maximumSize">
                      <size>
                       <width>75</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>从文件读取</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_16" native="true">
                  <layout class="QGridLayout" name="gridLayout_17">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton_5">
                     <property name="maximumSize">
                      <size>
                       <width>75</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string>手动框取</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_17" native="true">
                  <layout class="QGridLayout" name="gridLayout_18"/>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="widget_6" native="true">
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QGraphicsView" name="graphicsView"/>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QGridLayout" name="gridLayout_8">
               <item row="0" column="0">
                <widget class="QGraphicsView" name="graphicsView_2"/>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <widget class="QWidget" name="widget_8" native="true">
              <layout class="QGridLayout" name="gridLayout_9"/>
             </widget>
             <widget class="QWidget" name="widget_9" native="true">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>70</height>
               </size>
              </property>
              <layout class="QGridLayout" name="gridLayout_10">
               <item row="0" column="0">
                <widget class="QSplitter" name="splitter_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <widget class="QWidget" name="widget_18" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_19">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton_6">
                     <property name="maximumSize">
                      <size>
                       <width>23</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_19" native="true">
                  <property name="maximumSize">
                   <size>
                    <width>61</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_20">
                   <item row="0" column="0">
                    <widget class="QPushButton" name="pushButton_7">
                     <property name="maximumSize">
                      <size>
                       <width>23</width>
                       <height>23</height>
                      </size>
                     </property>
                     <property name="text">
                      <string/>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="widget_21" native="true">
                  <layout class="QGridLayout" name="gridLayout_21">
                   <item row="0" column="0">
                    <widget class="QSlider" name="horizontalSlider">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1337</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen_Image"/>
    <addaction name="actionOpen_Video"/>
    <addaction name="actionOpen_Camera"/>
    <addaction name="separator"/>
    <addaction name="actionOpen_OTB_set"/>
    <addaction name="actionOpen_VOT_set"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_Image">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionOpen_Video">
   <property name="text">
    <string>Open Video</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
  <action name="actionOpen_OTB_set">
   <property name="text">
    <string>Open OTB set</string>
   </property>
  </action>
  <action name="actionOpen_VOT_set">
   <property name="text">
    <string>Open VOT set</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
